airflow/models/connection.py

```diff
@@ -16,6 +16,8 @@
 # specific language governing permissions and limitations
 # under the License.
 
+# pylint: disable=R0913
```

Comment:
Can you explain it?

Reply:
Rule R0913 of `pylint` expected functions and methods to have utmost 10 arguments. The `__init__` method of `Connection` class already had 10 arguments. I added `description` which made it 11. So, had to disable rule R0913 and I followed the convention [here](https://github.com/apache/airflow/blob/9549274d110f689a0bd709db829a4d69e274eed9/airflow/providers/google/cloud/operators/dlp.py#L19). 

Would it be better to replace `# pylint: disable=R0913` with `#pylint: disable=too-many-arguments` and add it at the end of the line `class Connection(Base, LoggingMixin):` instead of disabling the rule for the whole file?

Reply:
> Would it be better to replace # pylint: disable=R0913 with #pylint: disable=too-many-arguments and add it at the end of the line class Connection(Base, LoggingMixin):

That would be preferable as others will now what exactly is disabled

Reply:
Done :) Added `#pylint: disable=too-many-arguments` to `__init__` method of Connection class.